docs/developer.md

```diff
@@ -35,12 +35,9 @@ This is recommended when working on features that are purely Armada specific or
 
 1. Get kind (Installation help [here](https://kind.sigs.k8s.io/docs/user/quick-start/))
     ```bash
-    GO111MODULE="on" go get sigs.k8s.io/kind@v0.5.1
+    GO111MODULE="on" go get sigs.k8s.io/kind@v0.11.1
     ``` 
-2. Create kind clusters (you can create any number of clusters)
-
-    As this step is using Docker, it will require root to run
```

Comment:
I don't think this requires root? At least not if you've done https://docs.docker.com/engine/install/linux-postinstall/?

Reply:
In practice it does require root on many machines. Maybe change to "it might require root (depending on how your machine is set up)" or something

Reply:
done